dir.props

```diff
@@ -81,8 +81,13 @@
   <PropertyGroup>
     <ValidatePackageVersions>true</ValidatePackageVersions>
     <ProhibitFloatingDependencies>true</ProhibitFloatingDependencies>
+
     <CoreFxExpectedPrerelease>rc4-24131-00</CoreFxExpectedPrerelease>
-    <CoreFxVersionsIdentityRegex>^(?i)((System\..*)|(NETStandard\.Library)|(Microsoft\.CSharp)|(Microsoft\.NETCore.*)|(Microsoft\.TargetingPack\.Private\.(CoreCLR|NETNative))|(Microsoft\.Win32\..*)|(Microsoft\.VisualBasic))(?&lt;!TestData)(?&lt;!System\.Data\.SqlClient)(?&lt;!System\.IO\.Compression)$</CoreFxVersionsIdentityRegex>
+    <CoreClrExpectedPrerelease>rc4-24131-00</CoreClrExpectedPrerelease>
+    <CoreLibTargetingPackExpectedPrerelease>rc4-24131-00</CoreLibTargetingPackExpectedPrerelease>
+    <ExternalExpectedPrerelease>rc4-24131-00</ExternalExpectedPrerelease>
+
+    <CoreFxVersionsIdentityRegex>^(?i)((System\..*)|(NETStandard\.Library)|(Microsoft\.CSharp)|(Microsoft\.NETCore\.Targets)|(Microsoft\.NETCore\.Platforms)|(Microsoft\.Win32\..*)|(Microsoft\.VisualBasic))(?&lt;!TestData)(?&lt;!System\.Data\.SqlClient)(?&lt;!System\.IO\.Compression)$</CoreFxVersionsIdentityRegex>
```

Comment:
We really do need to switch the update to take the package list from the versions repo instead of using a regex. I'm fine with this for now but can you please file a tracking issue to switch to the package list?